packages/embarkjs/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
LICENSE can probably be at the root, no need to replicate it

Reply:
Yes, `lerna publish` will [make sure the root LICENSE gets packed into the tarball](https://github.com/lerna/lerna/blob/master/commands/publish/index.js#L524) of any package that doesn't have its own LICENSE file.  I wasn't aware of that behavior because the explanation of it was removed from lerna-publish's README, but the behavior was added in https://github.com/lerna/lerna/pull/1465.